source/concepts/environments/problem-and-solution.html.md.erb

```diff
@@ -30,120 +30,120 @@ For example, if we had an account for `fictional-business-unit-production` and s
+Access keys are leaked for an IAM user in `fictional-business-unit-production`. The IAM user has an `AdministratorAccess` policy attached. A malicious attacker with these keys now has the ability to access both sensitive datasets: `dataset A` and `dataset B`.
 
-#### High-availability example
+#### Example: Loss of availability
 
 `fictional-business-unit-production` had only ever configured their EC2 instances to run in one AZ. If this AZ goes down, all of their applications do too.
 
-### Issue 2: team isolation
+### Issue 2: Team isolation
 
 Similar to Issue 1, team isolation becomes extremely difficult to maintain if all applications in their SDLC stage are within one account.
 
 Imagine `team-a` works on `example-a` and `team-b` works on `example-b`, both of which fall under `fictional-business-unit`.
 
-Everything `team-a` does can be seen by `team-b` even if you use a well-defined [attribute-based access control](https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction_attribute-based-access-control.html) (ABAC). Using ABAC requires resources to be tagged, and it has no effect on untagged resources. ABAC also doesn't work for all resources in AWS.
+Everything `team-a` does can be seen by `team-b` even if you use a well-defined [attribute-based access control (ABAC)](https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction_attribute-based-access-control.html). Using ABAC requires resources to be tagged, and it has no effect on untagged resources.ABAC also doesn't work for all resources in AWS.
 
-If `team-a` forgets to tag something, or creates a resource that doesn't support tagging during creation, it's interactable by who has access to that AWS account.
+If `team-a` forgets to tag something, or creates a resource that doesn't support tagging during creation, anyone who has access to that AWS account will be able to interact with it.
 
-#### An extreme misclicking example
+#### Example: Human error (misclick)
 
-`team-a` notices an issue with one of their `production` EC2 instances. They try to terminate it, and notice that they've just terminated a `production` instance that belonged to `team-b`, which disrupts `team-b`'s workflow whilst they redeploy their application. It takes time and money to reprovision these resources.
+`team-a` notices an issue with one of their `production` EC2 instances. They try to terminate it, and notice that they've just terminated a `production` instance that belonged to `team-b`, which disrupts `team-b`'s workflow whilst they redeploy their application. It takes time and money to reprovision these resources, and may affect application users.
 
-### Issue 3: billing
+### Issue 3: Billing
 
-In a shared AWS account for a business unit, untaggable billable items such as Data Transfer become impossible to attribute to an application.
+In a shared AWS account for a business unit, untaggable billable items such as Data Transfer cannot easily be attributed to a particular application.
```

Comment:
> untaggable billable items such as Data Transfer cannot easily be attributed to a particular application

Is this still the case (I don't have access to any cost allocation billing report to check)? https://repost.aws/questions/QUSiF9OHXiSqmYWZ4BbnAKeA/data-transfer-tagging

Reply:
Not entirely sure. This may be a comment from an historic perspective. It could now be, with tags and other items included, they can be. As you say access to specifics would be best to check this.